src/tf/parse_relu6.cpp

```diff
@@ -26,7 +26,8 @@ struct parse_relu6 : op_parser<parse_relu6>
             info.add_instruction(make_op("multibroadcast", {{"out_lens", input_lens}}), min_val);
         max_val =
             info.add_instruction(make_op("multibroadcast", {{"out_lens", input_lens}}), max_val);
-        return info.add_instruction(make_op("clip"), args.front(), min_val, max_val);
+
+        return info.add_common_op("clip", args[0], min_val, max_val);
```

Comment:
since we are using `add_common_op` you can remove multi-broadcasts on `max_val` and `min_val`. `add_common_op` would handle it for you.